test/Templates.Test/Helpers/AspNetProcess.cs

```diff
@@ -63,10 +63,15 @@ public AspNetProcess(ITestOutputHelper output, string workingDirectory, string p
                     .WaitForExit(assertSuccess: true);
             }
 
-            var envVars = new Dictionary<string, string>
+            var envVars = new Dictionary<string, string>();
+            if (noHttps)
```

Comment:
I think this is on the wrong PR, isn't it?

I've realized that the first url is HTTP, not HTTPS, so I think we are good even without this change. Sorry I sent you in the wrong direction.